packages/ui/icons/network/naked/PendulumNaked.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+
+//TODO change
```

Comment:
Yes and no. The svg from [here ](https://pendulumchain.org/brand-assets) had a different format to how the ones from amplitude was defined. It works, but I was unsure they are correct from the design standpoint.

Reply:
Which logos did you use (ie. what's the name of the files in the brand assets folder)?

Reply:
I used the ones in Brand Assets, `LOGO/SVG/Pendulum - RGB_Logomark - dark on bright` for `PendulumCircle` and `Pendulum - RGB_Logomark - dark` for the `Naked` one.

Reply:
I see. As far as I can tell, the 'circle' icons all have an some extra circle-shaped background. I think the 'LOGO/SVG/Pendulum - RGB_Logomark - dark on bright' is indeed the correct fit but we somehow need to make the background be circle shaped instead of rectangular. I checked the files you added using the visualizer you linked to, and it seems to me like the 'naked' and 'circle' logo we have now are the same (?).
We can reach out to Klaus to quickly provide us with this custom icon.

Besides that, we still need to pass the `{...props}` into the svg like all the other icons do.